Gemfile

```diff
@@ -4,11 +4,13 @@ source "https://rubygems.org"
 
 ruby RUBY_VERSION
 
-gem "decidim", "0.21.0"
-gem "decidim-consultations", "0.21.0"
+DECIDIM_VERSION = { git: "https://github.com/decidim/decidim", tag: "release/0.22-stable" }
```

Comment:
We should create an issue to change this to 0.22.X whenever they publish the latest patch.